docs/blocks/index.md

```diff
@@ -38,7 +38,7 @@ When a block is registered, *only* a block is registered. The block does not aut
 
 #### Optionally Registering Blocks
 
-In the past there have been several mods that have allowed users to disable blocks/items in a configuration file. However, you shouldn't do this. There is no limit on the amount of blocks that can be register, so register all blocks in your mod! If you want a block to be disabled through a configuration file, you should disable the crafting recipe. If you would like to disable the block in the creative tab, use a `FeatureFlag` when building the contents within [`CreativeModeTabEvent$BuildContents`][creativetabs].
+In the past there have been several mods that have allowed users to disable blocks/items in a configuration file. However, you shouldn't do this. There is no limit on the amount of blocks that can be register, so register all blocks in your mod! If you want a block to be disabled through a configuration file, you should disable the crafting recipe. If you would like to disable the block in the creative tab, use a `FeatureFlag` when building the contents within [`BuildCreativeModeTabContentsEvent`][creativetabs].
```

Comment:
`[creativetabs]` links needs to be updated. Matching upstream: [MinecraftForge/Documentation@`1.19.x...1.20.x`#diff-9b91547ac8](https://github.com/MinecraftForge/Documentation/compare/1.19.x...1.20.x#diff-9b91547ac8a61a49f08bca4e44988b3f80080288c9ccb2ed5a18a0abc2a475c1R47).